packages/core/src/providers/42-school.ts

```diff
@@ -186,17 +186,17 @@ export interface FortyTwoProfile extends UserData, Record<string, any> {
  *
  * #### Configuration
  *```ts
- * import { Auth } from "@auth/core"
- * import FortyTwoSchool from "@auth/core/providers/42-school"
+ * import Auth from "@auth/core"
+ * import FortyTwo from "@auth/core/providers/42-school"
  *
  * const request = new Request(origin)
  * const response = await Auth(request, {
  *   providers: [
- *     FortyTwoSchool({
- *       clientId: FORTY_TWO_SCHOOL_CLIENT_ID,
- *       clientSecret: FORTY_TWO_SCHOOL_CLIENT_SECRET,
- *     }),
- *   ],
+       FortyTwo({
+         clientId: 42_SCHOOL_CLIENT_ID,
+         clientSecret: 42_SCHOOL_CLIENT_SECRET
+       })
+     ],
```

Comment:
```suggestion
 *     FortyTwo({
 *       clientId: FORTY_TWO_CLIENT_ID,
 *       clientSecret: FORTY_TWO_CLIENT_SECRET,
 *     }),
 *   ],
```

Reply:
I would not recommend to do so unless you are willing to replace the `42-School` by `FortyTwo` everywhere else in the code base, even the file names 
Rationale: Using `Forty_Two` (or `FortyTwo` for consistency with current provider name) can be a good idea to remove numbers from the naming scheme. However, the 42-school reference is used across the codebase.
An example
```ts
//docs/pages/data/manifest.json
  ...
  "providersOAuth": {
    "github": "GitHub",
    "google": "Google",
    "twitter": "Twitter",
    "apple": "Apple",
    "discord": "Discord",
    "auth0": "Auth0",
    "facebook": "Facebook",
    "keycloak": "Keycloak",
    "42-school": "FortyTwo", // <-- // used in docs/components/OAuthProviderInstructions/content/index.tsx, for example
```

I am happy to do it myself. @ndom91 which way should I proceed?

Reply:
I agree.

But this is TypeScript code, so we can't use variables that start with a number.

And at least asterisks are required.

Reply:
Yeah so I think refactoring this to use `FortyTwo` everywhere makes sense. But as yuo rightly pointed out, it is used all over the place. 

We'd really appreciate if yuo cuold go through this with a fine tooth comb and make sure its not breaking anything :blush: